src/print/native.rs

```diff
@@ -140,26 +106,13 @@ impl FormatExt for Value {
             V::BigInt(v) => prn.const_scalar(format_bigint(v.into())),
             V::Decimal(v) => prn.const_scalar(format_decimal(v.into())),
             V::Bool(v) => prn.const_scalar(v),
-            V::Datetime(t) => prn.typed("datetime", format_rfc3339(*t)),
-            V::LocalDatetime(dt) => {
-                match TryInto::<NaiveDateTime>::try_into(dt) {
-                    Ok(naive) => prn.typed("cal::local_datetime",
-                        naive.format_with_items(DATETIME_FORMAT.iter())),
-                    Err(e) => prn.error("cal::local_datetime", e),
-                }
-            }
-            V::LocalDate(d) => {
-                match TryInto::<NaiveDate>::try_into(d) {
-                    Ok(naive) => prn.typed("cal::local_date",
-                        naive.format_with_items(DATE_FORMAT.iter())),
-                    Err(e) => prn.error("cal::local_date", e),
-                }
-            }
-            V::LocalTime(t) => {
-                prn.typed("cal::local_time",
-                    Into::<NaiveTime>::into(t)
-                        .format_with_items(TIME_FORMAT.iter()))
-            }
+            V::Datetime(t) => prn.typed("datetime", format!("{:?}", t)),
+            V::LocalDatetime(t)
+            => prn.typed("std::local_datetime", format!("{:?}", t)),
```

Comment:
```suggestion
            => prn.typed("cal::local_datetime", format!("{:?}", t)),
```

Reply:
Yeah, thanks. That's some attention blindness. There are no prefixes for `std::` types here so I definitely meant `cal::` :)